data/pages-content/terms.md

```diff
@@ -130,39 +122,38 @@ difference between the discounted ticket and the standard ticket price.
 Conference tickets are not subject to the
 [EU regulations on distance selling and the 14-day cooling off period](https://europa.eu/youreurope/citizens/consumers/shopping/guarantees-returns/index_en.htm).
 Outside the above refund policy, we do not provide refunds on ticket purchases
-after the Refund End Date.
+after the Refund End Date, though we will always consider your particular circumstances.
 
 ## 5. Data Protection
 
-The Organizers are considered data controllers under the terms of the
+The Organisers are considered data controllers under the terms of the
 [Regulation (EU) No. 2016/679 (GDPR)](https://eur-lex.europa.eu/legal-content/EN/LSU/?uri=celex:32016R0679).
 
 Please refer to our [Data Protection Policy](https://www.europython-society.org/privacy/) for details on how your data is
 processed.
 
-All attendees, sponsors and other participants of the conference have to declare
-their consent to the above privacy policy.
+All attendees, sponsors and other participants of the conference acknowledge and consent to the above privacy policy.
 
 ### Right of Denial
 
-The Organizers hereby notifies that the Participant has the right to deny the
+The Organisers hereby notifies that the Participant has the right to deny the
 storage and use of their personal information for advertising and marketing use.
 If desired, participants may revoke their consent to the use of their personal
 information for the foregoing reasons, at any time. It is sufficient to do so by
 email to: [privacy@europython.eu](mailto:privacy@europython.eu) or alternatively
-writing a letter to the Organizers.
+writing a letter to the Organisers.
 
 ### Recordings and Photography
 
 Participants may be photographed, filmed, or their voices recorded during the
-Event. Such recordings may be published by the Organizers and/or used for
+Event. Such recordings may be published by the Organisers and/or used for
 marketing purposes. Participation in the Event is conditional on the
-Participant’s consent to this.
+Participant’s consent to this, though Organisers will make an effort to respect the wish to to be photographed by the Participants.
```

Comment:
I would also leave this as is. If we go with the lanyards as last year, it will be clear that we make the effort, and if we spell it out in the T&C any misses risk giving us more bother.